### PR TITLE
Add a Makefile

### PR DESCRIPTION
**Pull request by tvoinarovskyi**
With this, users which are less familiar with the project can easily build, generate docs and run tests.

### CODE REVIEW

**Comment by tvoinarovskyi** [comment]
@dpkp @jeffwidman Any commands you want to add?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Some simple testing tasks (sorry, UNIX only).
+
+FLAGS=
+KAFKA_VERSION=0.11.0.0
```
Comment (dpkp) [inline Makefile]: can update to 0.11.0.1 now

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+SCALA_VERSION=2.11
+
+setup:
+	pip install -r requirements-dev.txt
```
Comment (dpkp) [inline Makefile]: probably want a virtualenv here, dont we? or perhaps --user ?

**Comment by dpkp** [comment]
might be nice to also include a command to fetch / install server artifacts via build_integration.sh

File: Makefile (hunk 2 above)
Comment (tvoinarovskyi) [reply Makefile]: I would prefer to assume the virtualenv is already installed and activated. There are a lot of different ways to do that, like pyvenv, virtualenv, virtualenvwrapper and so on.

Comment (tvoinarovskyi) [reply Makefile]: I can change the name to something like `setup-local`

**Pull request merged**